lib/rubocop/cop/rails/active_record_override.rb

```diff
@@ -48,6 +47,11 @@ def on_def(node)
 
         private
 
+        def active_model?(parent_class)
+          parent_class && %w[ApplicationRecord ActiveModel::Base]
+            .include?(parent_class.const_name)
```

Comment:
Originally, I had just done: 

```diff
-                        .include?(parent_class.const_name)
+                        .include?(parent_class&.const_name)
```

And that led to:
```console
lib/rubocop/cop/rails/active_record_override.rb:42:49: E: Lint/Syntax: unexpected token error
(Using Ruby 2.2 parser; configure using TargetRubyVersion parameter, under AllCops)
                          .include?(parent_class&.const_name)
```

----

So then changed it to:
```diff
-          return unless %w[ApplicationRecord ActiveModel::Base]
+          return unless parent_class && %w[ApplicationRecord ActiveModel::Base]
                           .include?(parent_class.const_name)
```

And that resulted in:
```console
lib/rubocop/cop/rails/active_record_override.rb:33:9: C: Metrics/AbcSize: Assignment Branch Condition size for on_def is too high. [17.03/17]
        def on_def(node) ...
```

So that's what got us to this extraction of `active_model?`